<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RicercaFumetto</class>
 <widget class="QWidget" name="RicercaFumetto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ricerca del Fumetto</string>
  </property>
  <widget class="QLineEdit" name="line_barra">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ricerca">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci il codice per la ricerca del fumetto:</string>
   </property>
  </widget>
  <widget class="QListView" name="lista_fumetti">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>571</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apri">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apri</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
